core/vm/contracts.go

```diff
@@ -1048,3 +1062,65 @@ func (c *bls12381MapG2) Run(input []byte) ([]byte, error) {
 	// Encode the G2 point to 256 bytes
 	return g.EncodePoint(r), nil
 }
+
+type remoteStaticCall struct{}
+
+func (c *remoteStaticCall) RequiredGas(input []byte) uint64 {
+	return params.RemoteStaticCallGas
```

Comment:
If we move to the non abi format, then we can have a function that charges based on the size of the data since it'll be easy to pull out the size, this is ok for now but def would be good to include when it comes to the gas metering thoughts

Reply:
Yeah we can definitely do that, something like `params.RemoteStaticCallGasMultiplier * len(input)` or something like that?

Reply:
As an update, I just changed it to that: `params.RemoteStaticCallGas * uint64(len(data))`